R/utilities-figures.R

```diff
@@ -301,7 +301,7 @@ createPlotsFromExcel <- function(plotGridNames = NULL, simulatedScenarios, obser
       individual = plotIndividualTimeProfile(dataCombined, plotConfigurationList[[plotId]]),
       population = {
         quantiles <- dfPlotConfigurations[dfPlotConfigurations$plotID == plotId, ]$quantiles
-        if (is.na(quantiles)) {
+        if (is.null(quantiles) || is.na(quantiles)) {
```

Comment:
If no column `quantiles` is found, `quantiles` is `NULL` and must be captured here.